logdna/logdna.py

```diff
@@ -325,6 +323,9 @@ def emit(self, record):
         self.buffer_log(message)
 
     def close(self):
+        # Close the flusher
+        self.close_flusher()
```

Comment:
close_flusher & start_flusher are called under lock except this place. 
harmless?

Reply:
It should be, yes, because close is only called by the python logging library when the application closes. Further, closing / cancelling the flusher [is a no-op if the flush timer thread has already started](https://docs.python.org/3/library/threading.html#threading.Timer.cancel).